pooch/tests/test_version.py

```diff
@@ -0,0 +1,9 @@
+"""
+Test the version.
+"""
+import pooch
+
+
+def test_version():
+    "Check there's a usable version number in the usual __version__"
+    assert pooch.__version__.startswith("v")
```

Comment:
```suggestion
    assert pooch.__version__.startswith("v")
    # Check that it's PEP440 compliant (will raise an exception otherwise)
    Version(pooch.__version__)
```